ui/js/modal/modalRewardApprovalRequired/view.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Modal } from "modal/modal";
+
+class ModalRewardApprovalRequired extends React.PureComponent {
+  render() {
+    const { closeModal, doAuth } = this.props;
+
+    return (
+      <Modal
+        isOpen={true}
+        contentLabel={__("Human Verification Required")}
+        onConfirmed={doAuth}
+        onAborted={closeModal}
+        type="confirm"
+        confirmButtonLabel={__("I'm Totally Real")}
+        abortButtonLabel={__("Nevermind")}
```

Comment:
"Never mind" are two words.

Reply:
It's two in this case, good catch. It's confusing for everyone :)